### DISCUSSION TITLE
Is there any way to focus a particular row which we are dragging over the other row for better UX.

### DISCUSSION THREAD

**Discussion by AdityaBhadange** [KevinVandy/material-react-table#346] [General]
I was wondering if we have a way to **focus** (e.g. Showing custom row border color or any custom CSS in general) on the target row while row dragging/ordering.
And also how to conditionally enable the focus on the row.

**Note** - I'm not talking about changing the row's opacity.

Thanks

**Comment by KevinVandy** [comment]
Yes, that's supposed to be happening. I'm guessing it broke after the border-collapse css rule was internally changed to fix a firefox bug recently.

**Reply by KevinVandy** [reply]
To fix for yourself:

```jsx
muiTableProps={{
  sx: {
    borderCollapse: 'collapse'
  }
}}
```

I'll see what can be done to fix it and keep row borders still showing in firefox soon.

**Reply by AdityaBhadange** [reply]
@KevinVandy Hey, thanks a lot for the response. Your solution fixed the issue.

**Comment by KevinVandy** [comment]
should be back to normal in v1.6.4
